content/doc/developer/publishing/releasing-cd.adoc

```diff
@@ -0,0 +1,149 @@
+---
+title: Setting up automated plugin release
+layout: developerguide
+references:
+- url: /jep/229
+  title: 'JEP-229: Continuous Delivery of Jenkins Components and Plugins'
+---
+
+== Prerequisites
+
+. Plugin must be 'incrementalized', see link:../../plugin-development/incrementals[incrementals], a new plugin created from the link:https://github.com/jenkinsci/archetypes/[archetypes] will already have this setup.
+. Enabled the continuous delivery flag in link:https://github.com/jenkins-infra/repository-permissions-updater/[repository permission updater] (RPU) for your plugin.
+
+
+[source,yaml]
+----
+cd:
+  enabled: true
+----
+
+Once that has been merged, start checking https://github.com/jenkinsci/your-plugin/settings/secrets/actions and you should soon see `MAVEN_TOKEN` and `MAVEN_USERNAME` appear under Repository secrets.
+
+== Release notes
+
+Next, if you already have Release Drafter configured, remove any tag-template override in `.github/release-drafter.yml`, and delete `.github/workflows/release-drafter.yml` if using GitHub Actions (or remove the app from the repo otherwise). If you have not yet configured Release Drafter, just create `.github/release-drafter.yml` containing only:
+
+[source,yaml]
+----
+_extends: .github
+----
+
+== Adding the GitHub action
+
+Create `.github/workflows/cd.yaml` as a copy of the link:https://github.com/jenkinsci/.github/blob/master/workflow-templates/cd.yaml[cd.yaml template].
+
+If this is your first time using a GitHub action on this project then navigate to the 'Actions' tab 
+in your GitHub repository and enable GitHub actions.
+
+If you have a `.github/dependabot.yml`, it is a good idea to add:
+
+[source,yaml]
+----
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: daily
+----
+
+== Pom modifications
+
+For a regular component whose version number is not that meaningful:
```

Comment:
Will need to call attention to the fact that you should have no expectation of explicitly picking version numbers in this case; was a source of confusion in the case of `token-macro`.

Reply:
is this what you meant? https://github.com/jenkins-infra/jenkins.io/pull/4341/files#r629353070

Reply:
Works.